tests/cypress/support/commands.ts

```diff
@@ -41,6 +41,7 @@ Cypress.Commands.add('requestFindNodeInnerHTMLByName', function (url: string, no
             .find(nodeName)
             .toArray()
             .map((el) => el.innerText)
+        JSON.stringify(nodes)
```

Comment:
This doesn't seem to be doing anything as return value not used

Reply:
You are right I will remove this (forgot to remove)